scripts/chief_deploy.py

```diff
@@ -51,6 +51,7 @@ def update_product_details(ctx):
 @task
 def update_locales(ctx):
     with ctx.lcd(os.path.join(settings.SRC_DIR, 'locale')):
+        ctx.local("dennis-cmd lint --errorsonly .")
```

Comment:
Awesome! That's a little different than what we did with SUMO and Input, but I think that's fine.

Does it make sense to change `locale/compile.sh` to die with a non-zero status code if compiling ever failed.